src/rules/banRule.ts

```diff
@@ -25,23 +25,26 @@ export class Rule extends Lint.Rules.AbstractRule {
         ruleName: "ban",
         description: "Bans the use of specific functions.",
         descriptionDetails: "At this time, there is no way to disable global methods with this rule.",
-        optionsDescription: "A list of `['object', 'method']` pairs which ban `object.method()`.",
+        optionsDescription: "A list of `['object', 'method', 'optional explanation here']` which ban `object.method()`.",
         options: {
             type: "list",
             listType: {
                 type: "array",
                 arrayMembers: [
                     { type: "string" },
                     { type: "string" },
+                    { type: "string" },
                 ],
             },
         },
-        optionExamples: [`[true, ["console", "log"], ["someObject", "someFunction"]]`],
+        optionExamples: [`[true, ["someObject", "someFunction"], ["someObject", "someFunction", "Optional explanation"]]`],
```

Comment:
We don't want both banned functions to be `someObject` and `someFunction`, can we change those terms for one of them?